pkg/minikube/node/start.go

```diff
@@ -379,9 +381,27 @@ func configureRuntimes(runner cruntime.CommandRunner, cc config.ClusterConfig, k
 		exit.Error(reason.InternalRuntime, "Failed runtime", err)
 	}
 
-	disableOthers := true
-	if driver.BareMetal(cc.Driver) {
-		disableOthers = false
+	// 87-podman.conflist cni conf potentially conflicts with others and is created by podman on its first invocation,
+	// so we "provoke" it here to ensure it's generated and that we can disable it
+	// note: using 'help' or '--help' would be cheaper, but does not trigger that; 'version' seems to be next best option
+	if co.Type == constants.CRIO {
+		_, _ = runner.RunCmd(exec.Command("sudo", "sh", "-c", `podman version >/dev/null`))
```

Comment:
Maybe not necessary, but could check for an error if somehow `podman` is not present?